pkgs/data/fonts/nerdfonts-patcher/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nerdfonts-patcher-unwrapped";
+  version = "v2.1.0";
+
+  # This repo is approximately 2.2GB in size, and the font patcher isn't available separately
+  # see this issue: https://github.com/ryanoasis/nerd-fonts/issues/484
+  # The fork mentioned in the issue could be used, but I don't know how updated it is and will be.
+  src = fetchFromGitHub {
+    owner = "ryanoasis";
+    repo = "nerd-fonts";
+    rev = version;
+    sha256 = "sha256-j0cLZHOLyoItdvLOQVQVurY3ARtndv0FZTynaYJPR9E=";
+  };
+
```

Comment:
You have to specify all dependencies of this package in buildInputs. In your case fontforge ... python itself is covered already in buildPythonApplication.

Reply:
I noticed that youtube-dl uses buildPythonPackage. Is there a difference between them?

Reply:
The difference is small and should behave the same, you can find the definition here:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L46